### PR TITLE
Updating documentation for nbclassic

### PR DESCRIPTION
**Pull request by RRosio**
Update readthedocs documentation WIP.

Work TBD:
- [x] Update homepage outline of documentation
- [x] Update development faq
- [ ] ~~Delete that documentation which is not applicable to nbclassic(?)~~

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -93,7 +93,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Jupyter Notebook'
+project = 'NbClassic'
```
Comment (echarles) [inline docs/source/conf.py:96]:
```suggestion
project = 'Jupyter NbClassic'
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
-====================
-The Jupyter Notebook
-====================
+===================================
+The Jupyter Notebook with NbClassic
```
Comment (echarles) [inline docs/source/index.rst:2]:
```suggestion
The Jupyter NbClassic
```

File: docs/source/nbclassic.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Launch directly:
+``> jupyter nbclassic``
+
+Alternatively, you can run Jupyter Server and visit the `/tree` endpoint:
+``> jupyter server``
+
+
+NbClassic in the Jupyter Ecosystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (echarles) [inline docs/source/nbclassic.rst:45]: Maybe refer to https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000

Comment (RRosio) [reply docs/source/nbclassic.rst:45]: Okay awesome! Thank you for the link, I will add it to the documentation too!

File: docs/source/nbclassic.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+on the same server.
+
+As nbclassic provides the static assets for notebook-v6.5.x, while
+having both installed should cause no issues, the user interface provided
+by these two packages will be the same. These UIs would be available in
+different servers.
+
+When using nbclassic and notebook <= v6.4.x you can expect that these UIs
+will not be only presented at different servers but may also differ as
+potential changes to the nblcassic UI will not be reflected in Notebook
```
Comment (Zsailer) [inline docs/source/nbclassic.rst:59]:
```suggestion
potential changes to the nbclassic UI will not be reflected in Notebook
```

**Comment by echarles** [comment]
@RRosio Please confirm when you'd like me to review this PR.

As discussed, you could add a page/section which highlights things people migrating form notebook to nbclassic should know or care about;

For example, https://github.com/jupyter/nbclassic/issues/140 depends https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/pull/141. Once https://github.com/Jupyter-contrib/jupyter_nbextensions_configurator/pull/141 will be merged and released, users will need to run the following command to activate the extension

```bash
pip install jupyter_nbextensions_configurator
jupyter nbextension install --sys-prefix --py jupyter_nbextensions_configurator --overwrite
jupyter nbextension enable --sys-prefix --py jupyter_nbextensions_configurator
jupyter serverextension install --sys-prefix --py jupyter_nbextensions_configurator --overwrite
jupyter serverextension enable --sys-prefix --py jupyter_nbextensions_configurator
```

**Comment by RRosio** [comment]
I did some reordering of sections here for readability and added a section for known issues. The Test Downstream job is failing but I opened another PR with a minimal change and it was still failing which leads me to think that the fail is not related to the changes in the PR.

**Review by echarles — APPROVED** [review]
LGTM Thx @RRosio

**Pull request merged**